nemo/collections/nlp/data/datasets/sgd_dataset/data_processor.py

```diff
@@ -87,7 +89,11 @@ def __init__(
         self._max_seq_length = self.schema_config["MAX_SEQ_LENGTH"]
 
         self.dial_files = {}
+        slots_relation_file = "slots_relation_list.np"
+        slots_relation_file = os.path.join(dialogues_example_dir, slots_relation_file)
```

Comment:
Could you please add a comment to explain why the`slot_relation_file` is needed?
you can replace 3 lines with self.slots_relation_file = os.path.join(dialogues_example_dir, "slots_relation_list.np")

Reply:
Done